packages/@adobe/spectrum-css-temp/components/dropdown/index.css

```
@@ -91,6 +91,10 @@ governing permissions and limitations under the License.
   margin-inline-start: var(--spectrum-selectlist-thumbnail-image-padding-x);
 }
 
+.spectrum-Avatar + .spectrum-Dropdown-label {
```

I know the icon classname is `.spectrum-Icon`, but that's a carry over of a mistake from multiple years ago, lets call this one
```suggestion
.spectrum-Dropdown-avatar + .spectrum-Dropdown-label {
```

helps avoid confusion on which CSS module it came from and prevents confusion on the classname being shared across the boundaries of the module